src/server/game/Handlers/CombatHandler.cpp

```diff
@@ -86,10 +86,12 @@ void WorldSession::SendAttackStop(Unit const* enemy)
 {
     WorldPacket data(SMSG_ATTACKSTOP, (8 + 8 + 4)); // we guess size
     data << GetPlayer()->GetPackGUID();
+    uint8 nowDead = enemy->isDead();
```

Comment:
This should probably be inside the IF below

Reply:
Because the if checks if enemy is null but this will use it before checking

Reply:
Good call, corrected.